frontend/js/containers/FullScreenPanel.jsx

```diff
@@ -130,6 +130,7 @@ class FullScreen extends React.Component {
         return (
             <SideQueryPanel
                 withMap={false}
+                hideSpatialFilter
```

Comment:
Is this intended? What does this option do?

Reply:
Yes it is. It is to hide spatial filter in the FullscreenScreen panel page where a spatial filter is not needed as we wont have a map to perform a spatial selection.